src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -37,7 +34,7 @@ public class AddCommandParser implements Parser<AddCommand> {
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE,
-                 PREFIX_EMAIL, PREFIX_DOB, PREFIX_ADDRESS, PREFIX_TAG, PREFIX_GENDER); // add gender prefix
+                 PREFIX_EMAIL, PREFIX_DOB, PREFIX_ADDRESS, PREFIX_GENDER); // add gender prefix
```

Comment:
Shall we just remove this comment too!
```suggestion
                 PREFIX_EMAIL, PREFIX_DOB, PREFIX_ADDRESS, PREFIX_GENDER); 
```